### PR TITLE
[Observability] Several fixes to the observability doc

### PR DESCRIPTION
**Pull request by scottsun94**
## Why are these changes needed?
Fix several issues in https://github.com/ray-project/ray/pull/35816

## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [x] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: doc/source/ray-observability/getting-started.rst [hunk 1]
```diff
@@ -274,6 +266,10 @@ See :ref:`System Metrics Page <system-metrics>` for available metrics.
 
 The Metrics view provides visualizations of the time series metrics emitted by Ray.
 
+You can select the time range of the metrics in the top right corner. The graphs refresh automatically every 15 seconds.
+
+There is also a convenient button to open the grafana UI from the dashboard. The Grafana UI provides additional customizability of the charts.
```
Comment (angelinalg) [inline doc/source/ray-observability/getting-started.rst:271]:
```suggestion
There is also a convenient button to open the Grafana UI from the dashboard. The Grafana UI provides additional customizability of the charts.
```

File: doc/source/ray-observability/user-guides/cli-sdk.rst [hunk 2]
```diff
@@ -49,7 +49,7 @@ Ray State CLI and SDK
 
 .. tip:: We'd love to hear your feedback on using Ray state APIs - `feedback form <https://forms.gle/gh77mwjEskjhN8G46>`_!
 
-Use Ray State APIs to access the current state (snapshot) of Ray through a CLI or Python SDK (developer APIs).
+Use Ray State APIs to access the current state (snapshot) of Ray through CLI or Python SDK (developer APIs).
```
Comment (angelinalg) [inline doc/source/ray-observability/user-guides/cli-sdk.rst:52]:
```suggestion
Use Ray State APIs to access the current state (snapshot) of Ray through the CLI or Python SDK (developer APIs).
```

File: doc/source/ray-observability/user-guides/cli-sdk.rst [hunk 3]
```diff
@@ -49,7 +49,7 @@ Ray State CLI and SDK
 
 .. tip:: We'd love to hear your feedback on using Ray state APIs - `feedback form <https://forms.gle/gh77mwjEskjhN8G46>`_!
```
Comment (angelinalg) [inline doc/source/ray-observability/user-guides/cli-sdk.rst:50]:
```suggestion
.. tip:: Provide feedback on using Ray state APIs - `feedback form <https://forms.gle/gh77mwjEskjhN8G46>`_!
```

File: doc/source/ray-observability/getting-started.rst [hunk 4]
```diff
@@ -340,13 +336,13 @@ The Logs view provides search functionality to help you find specific log messag
...
+If the Ray Job is submitted by the :ref:`Ray Job API <jobs-quickstart>`, the Job logs are available from the Dashboard. The log file follows the following format; ``job-driver-<job_submission_id>.log``.
 
 .. note::
 
   If the driver is executed directly on the head node of the Ray Cluster (without the job API) or run with :ref:`Ray client <ray-client-ref>`, the driver logs are not accessible from the dashboard. In this case, see the terminal output to view the driver logs.
 
-**Task and Actor Logs (worker logs)**
+**Task and Actor Logs (Worker logs)**
 
 Task and actor logs are accessible from the :ref:`task and actor table view <dash-workflow-state-apis>`. Click the "Log" button.
```
Comment (angelinalg) [inline doc/source/ray-observability/getting-started.rst:347]:
```suggestion
Task and Actor logs are accessible from the :ref:`task and actor table view <dash-workflow-state-apis>`. Click the "Log" button.
```

File: doc/source/ray-observability/getting-started.rst [hunk 5]
```diff
@@ -340,13 +336,13 @@ The Logs view provides search functionality to help you find specific log messag
 
 **Driver logs**
 
-If the Ray Job is submitted by the :ref:`Ray job API <jobs-quickstart>`, the Job logs are available from the Dashboard. The log file follows the following format; ``job-driver-<job_submission_id>.log``.
+If the Ray Job is submitted by the :ref:`Ray Job API <jobs-quickstart>`, the Job logs are available from the Dashboard. The log file follows the following format; ``job-driver-<job_submission_id>.log``.
 
 .. note::
 
   If the driver is executed directly on the head node of the Ray Cluster (without the job API) or run with :ref:`Ray client <ray-client-ref>`, the driver logs are not accessible from the dashboard. In this case, see the terminal output to view the driver logs.
```
Comment (angelinalg) [inline doc/source/ray-observability/getting-started.rst:343]:
```suggestion
  If you execute the driver directly on the head node of the Ray Cluster (without the Job API) or run with :ref:`Ray client <ray-client-ref>`, the driver logs are not accessible from the dashboard. In this case, see the terminal output to view the driver logs.
```

**Review by angelinalg — COMMENTED** [review]
some minor nits

**Review by rkooo567 — APPROVED** [review]

**Review by angelinalg — APPROVED** [review]

**Pull request merged**